invisible_cities/io/mcinfo_io_test.py

```diff
@@ -0,0 +1,146 @@
+import os
+import numpy  as np
+import tables as tb
+
+from .  mcinfo_io import load_mchits
+from .  mcinfo_io import load_mcparticles
+from .  mcinfo_io import load_mcsensor_response
+from .  mcinfo_io import mc_info_writer
+from .  mcinfo_io import read_mcinfo_evt
+
+from .. reco.tbl_functions import get_mc_info
+
+from pytest import mark
+parametrize = mark.parametrize
+
+
+@mark.serial
+@parametrize('skipped_evt, out_filename',
+                  ((0, 'test_mcinfo_skip_evt0.h5'), (1, 'test_mcinfo_skip_evt1.h5')))
```

Comment:
alignment. Put each set of parameters in a different line.

Reply:
Also, I would use `mark.parametrize` instead of
```
parametrize = mark.parametrize
@parametrize
```

because we use `mark.` anyway with `serial`.

Reply:
I disagree. Not strongly, but I disagree. The only reason we use =mark.parametrize= at all, is that it is impossible to import =parametrize= from =pytest.mark=, but morally speaking that's what we would want to do. =parametrize = mark.parametrize= is the next best thing.

Reply:
Ok. I don't have a strong opinion here, I just like the symmetry. I do also get the feeling that marking something with `serial` or with `parametrize` have different meanings.

Reply:
Symmetry would be an argument for `serial = mark.serial` :-)

But, now that you mention it, I agree that `mark.serial` somehow feels better than just `serial` while the opposite is true for `parametrize`.